src/Markdown.hs

```diff
@@ -26,14 +16,23 @@ instance ToMarkup Markdown where
     -- | Sanitized by default
     toMarkup = handleError . markdownToHtml
 
-strippedText :: ToMarkup a => a -> Text
-strippedText = pack . runLA
-    (   arr toHtml
-    >>> arr renderMarkup
-    >>> hread
-    //> getText
-    >>. concat
-    )
+toPandoc :: Markdown -> Pandoc
+toPandoc m = either mempty id $ parse m
```

Comment:
I tried that but couldn't figure it out! I'm probably going to merge this as-is, but I'd love to see how this can be made point free (like, reasonably without gettin weird with it)